src/Runtime/Runtime/PublicAPI/Interop/JavascriptCallback.cs

```diff
@@ -0,0 +1,78 @@
+﻿
+/*===================================================================================
+*
+*   Copyright (c) Userware/OpenSilver.net
+*
+*   This file is part of the OpenSilver Runtime (https://opensilver.net), which is
+*   licensed under the MIT license: https://opensource.org/licenses/MIT
+*
+*   As stated in the MIT license, "the above copyright notice and this permission
+*   notice shall be included in all copies or substantial portions of the Software."
+*
+\*====================================================================================*/
+
+
+using System;
+
+namespace CSHTML5.Internal
+{
+    public class JavascriptCallback : IDisposable
```

Comment:
For now I think it is better to use this class internally as its API will likely change in the future. When it will be more stable, we can change it back to public.
```suggestion
    internal class JavascriptCallback : IDisposable
```

Reply:
Good point! I am constantly forgetting that we develop the public tool, and we can not easily change interfaces later :(